### ISSUE TITLE
Issue - sprues too narrow for Choc v1 buttons

### ISSUE THREAD

**Issue by fudge01010** [TheTrainGoes/GP2040-Projects#13]
Looking to get some of the choc v1 buttons printed at JLCPCB, and they've come back requesting an increase in the sprue diameter - increased to 1.5mm for more reliable printing:

> As shown below, the connections of the models are too small and thin , so there will be high risk of broken and lost during production ...
>  ... could you please kindly increase the connections to 1.5 mm for better print effect ？

I noticed in #11 that you increased the sizes to 1.5mm - if you could do the same for these it would be appreciated :D 

Thanks dev!

**Comment by TheTrainGoes** [comment]
Hi @fudge01010 ,

These should be just fine to print as 1mm but I will get the files updated when I have a chance.

**Comment by fudge01010** [comment]
Hi again @TheTrainGoes !

Just as a note, JLC are very paranoid in printing this file as is:

> Good day.
> 
> Although you have confirmed to take the risk to print the weak connection，but JLC suggests you place order for the single part instead of connected part.
> 
> Usually, we will place the single model in 45 degree for better printing result, but can only place the connected parts in horizontal or vertical way for printing.
> 
> We have noticed that there are risk of size deviation in z-axis, deformation, layer textures line on the part, and the connection areas of the parts could be easily damaged and get lost for the penalized parts due to the weak connections.
> 
> The wall thickness of the connection is too thin, which is less than minimum requirement 1.5mm and has risks of crack, loss and damage both in printing and post process., If these risks occurred and parts break apart, shall we proceed with the shipment directly ?
> 
> Would you like to take the risk to proceed as it is?
> 
> Or you would like to modify file and replace file?
> 
> Your early reply will be highly appreciated !

I'd selected SLA w/ 9000R resin as per [this comment](https://github.com/jfedor2/flatbox/issues/4#issuecomment-1465325269) on the flatbox repo - should I select a different resin (or printing technology)?

I had a crack at editing the STL's directly in meshmixer to increase the sprue thicknesses, but didn't have much luck (the model ended up non manifold and very irregular, and still with thin walls in parts). Not sure what tool you designed these parts with, but I'm happy to attempt to edit them if you're willing to share the source files?

Thanks again!

**Comment by TheTrainGoes** [comment]
I made these in TinkerCAD off of existing models from various sources.  

Overall JLC has gone downhill in terms of quality while also raising their prices.  I have printed a lot of these with the 1mm sprue and never had any major issues, but I do recommend getting some extras.  

If you know anyone locally that can make them for you I would do that as the cost to have them done from JLC now is super high.
